remedy.lic

```diff
@@ -82,6 +82,7 @@ class Remedy
     waitrt?
     case bput(command,
               'You need another splash of water',
+              'You need another splash of alcohol',
```

Comment:
Are there any more liquids than water and alcohol? If so, it might be worth the time to future-proof this and match against `'You need another splash of (.*)'`